.github/workflows/ci.yml

```diff
@@ -0,0 +1,98 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+      - "dependabot/**"
```

Comment:
Is your intention to run build and lint if a new dependabot or snyk branch is opened? 

If so, we could alternatively configure this by targeting **all PRs to develop branc**h; this would account for all _dependabot -> develop_ and _snyk -> develop_ PRs, as described in the previous review comment.

Reply:
addressed in the prev comment!